docs/02-usage/02-framework.md

```diff
@@ -78,25 +78,36 @@ components like a footer.
 ### Themes
 
 Protocol supports [multiple brand themes](/docs/fundamentals/brand-themes), namely
-the Mozilla and Firefox brands. Themes work by defining sets of variables in Sass
-maps in the `/includes/_themes.scss` file, and those variables are fetched via
-the `get-theme()` function. That function determines which set of variables to
-draw from based on the declared brand theme, thus any themeable property in CSS
-can have different values for different brands.
+the Mozilla and Firefox brands. Themes work by declaring a set of CSS Custom Properties
+on the document's root element (`:root`) in `/includes/_themes.scss` and are retrieved
+using the `var()` function. The custom properties will be updated depending on what theme is declared.
```

Comment:
_(non-blocking, just noting for a potential future docs update)_
This is a very exciting change because we're not compiling away sass variables with internal functions anymore. We're using CSS inheritance. This allows end users more flexibility if the pre-defined Protocol themes don't apply (i.e. they are early days in the branding journey and want to experiment before creating an official Protocol theme).

We could provide a list of themeable CSS properties and demo how a user can define their own theme by declaring these properties _after_ the theme declarations

```css
/* user-lib.scss */
@use '~@mozilla-protocol/core/protocol/css/includes/themes';
/* user-defined theme values */
:root {
    --background-color-inverse: #{$color-green-80};
}
```
![Screenshot 2023-02-23 at 12 01 02 PM](https://user-images.githubusercontent.com/19650432/220900123-ba7fe59b-2f1a-4c56-8464-f7e9edee76b9.png)

Reply:
such a good idea!